main.php

```diff
@@ -55,6 +55,7 @@
 /**
  * Include the defines that set BASE_PATH, etc
  */
+require_once('core/Functions.php');
```

Comment:
It needs removing; the support file should be included immediately preceding the function call.